### ISSUE TITLE
three lost relations

### ISSUE THREAD

**Issue by ShellingFord221** [thunlp/OpenNRE#130]
Hi,
   There are 56 relations in sentences, but 53 in train.txt. In issue 121 you said that "There are no sufficient data for certain relations, so we just discard these relations and instances of those relations will be treated as NA during training."However, I find that three lost relations are not rare:
/business/company/industry: 6 sentences
/people/ethnicity/includes_groups: 7 sentences
/people/ethnicity/people: 169 sentences

   As a contrast, there are 4 relations whose sentences are only 1. They are:
/location/fr_region/capital
/business/shopping_center/owner
/business/shopping_center_owner/shopping_centers_owned
/location/mx_state/capital

   So why do you delete frequent relations and leave relations which only have 1 sentences?

**Comment by Spico197** [comment]
Hi, there. I got the same question. And I also got some new relations in `test.json`, which not appear in the training set. For this dataset, maybe your method is a good way to train, but it is still hard to cover all the relations, and that's why there are so many NA relations.

**Comment by gaotianyu1350** [comment]
Yes in the original dataset from [Riedel et al. 2010](https://link.springer.com/content/pdf/10.1007%2F978-3-642-15939-8_10.pdf) there are some relations that appear in the test set but not in the training set. Though we do not know the reason, we simply follow the setting since there are already many papers use this dataset and for model comparison, we have not done any changes.

Actually, NYT10 has many problems: not only the confusing relation division but the data distribution is extremely unbalanced. Most relation facts are covered by the top 5 relations. Recently we are going to release a new distant dataset based on Wikipedia, which will be clean and more balanced.
